src/compiler/checker.ts

```diff
@@ -15917,6 +15917,7 @@ namespace ts {
             //     export_opt   AmbientDeclaration
             //
             if (node.kind === SyntaxKind.InterfaceDeclaration ||
+                node.kind === SyntaxKind.TypeAliasDeclaration ||

```

Comment:
Even though it's not part of the spec, can you amend the grammar in the comment to reflect the change, and then add a `// TODO: The spec needs to be amended to reflect this grammar.` below the grammar?